src/api/types/personregisterTypes.ts

```diff
@@ -7,9 +7,9 @@ import {
 
 export type Skjermingskode = 'INGEN' | 'DISKRESJONSMERKET' | 'EGEN_ANSATT';
 export type ReadableSkjermingskode =
-  | 'ingen'
-  | 'diskresjonsmerket'
-  | 'egen ansatt';
+  | 'Ingen'
+  | 'Diskresjonsmerket'
+  | 'Egen ansatt';
 export type ReadableSkjermingskodeMap = Record<
   Skjermingskode,
   ReadableSkjermingskode
```

Comment:
Synes det blir mer forvirrende med en `type` og `Record`, og en egen funksjon ute i `personDataUtil` som bruker dette. Hva med en helt enkel funksjon som tar in enum og gir tekst? 😊
```javascript
export function toReadableSkjermingkode(
  skjermingskode: Skjermingskode
): string {
  switch (skjermingskode) {
    case 'INGEN':
      return 'Ingen';
    case 'DISKRESJONSMERKET':
      return 'Diskresjonsmerket';
    case 'EGEN_ANSATT':
      return 'Egen ansatt';
  }
}
```

Reply:
Enig i at det som ligger her var litt kronglete satt opp. Jeg valgte å ikke se altfor mye på det 😬